uiplib/gui/gallery.py

```diff
@@ -42,7 +44,12 @@ def set_image(self, imagePath):
                                 label="Blur", showvalue=0)
             self.slider.pack()
         self.image = UipImage(imagePath)
-        self.show_image(self.image)
+        im = Image.open(imagePath)
+        try:
+            im.load()
```

Comment:
@abhsag24 If the approach needs to be different i.e. changes are to be made in a different file, then I think it is better if the PR is closed.